CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support lists of files and folders as input to the `embed` and `train` commands.
 - Faster dataset initialization with SLURM and Windows.
 - Model is exported after every n-th epoch instead of only at the end. Default n=1
+- Add DistillationV2 to the methods.
```

Comment:
We should also mention that it is now the default distillation method that is selected with `method="distillation"`. For the old method `method="distillationv1"` must be used.